templates/partials/scripts.hbs

```diff
@@ -28,9 +28,9 @@
 <script src='/assets/js/docs.js'></script>
 <script src='/assets/js/ga.js'></script>
 <script src='/assets/js/device-animation.js'></script>
+<script src='/assets/js/firmwareversions.js'></script>
```

Comment:
Would it be possible to load this script on only the pages you need instead of on every one? Sometimes weird JS things happen when some jQuery expects content that isn't there